samples/0_Intro/hcc_dialects/vadd_hc_am.cpp

```diff
@@ -26,8 +26,8 @@ THE SOFTWARE.
 // which can only be used on the device.  The programmer has full control
 // over when data is copied.
 
-#include <hc.hpp>
-#include <hc_am.hpp>
+#include <hc/hc.hpp>
```

Comment:
Issue with this change is that if HCC is installed using ```make install```, there is no hc folder under hcc/include. So unless the only supported way of using HCC is to install using deb/rpm packages, this change is not acceptable. Alternatively HCC needs to fix ```make install``` to create hcc/include/hc and place all the include files under it.

Reply:
Of course it is acceptable m8, it's just that these silly interop examples will be temporarily disabled or ignored. Nobody uses this anyway. This has nothing to do with debs (that creates different path anyway), but rather pertains to how the HC headers will be exposed going forward, i.e. they will be in hc/, with the legacy non-prefixed stuff existing only for HIP's perusal.

Reply:
Okay that is acceptable.

Reply:
I've refactored this to have the new includes under a guard. I do believe we may want to eventually remove the samples, but they'll stay usable during the transition.